src/main.rs

```diff
@@ -482,9 +484,9 @@ async fn uuid_get(
     let lock = OUT.read().await;
     let exec = lock.get(&uuid).ok_or_else(redirect::workload_not_found)?;
     let claims = claims.map_err(|e| e.redirect_response())?;
-    let user = claims.subject().to_string();
+    let user = claims.github();
 
-    if exec.lock().await.user != user {
+    if Some(exec.lock().await.user.as_str()) != user {
```

Comment:
I'd expect to see an `ok_or` here https://doc.rust-lang.org/nightly/core/option/enum.Option.html#method.ok_or with clear failure case handling, but I don't mind this as a quickfix.
This code would benefit from some refactoring.